src/WebExploitation/FileUploads/Content-type restriction.md

```diff
@@ -0,0 +1,89 @@
+# File Uploads: Content-type restriction
+
+## Introduction
+In the wild, it's unlikely that you'll find a website that has no protection against file upload attacks like we saw in the previous lab. But just because defenses are in place, that doesn't mean that they're robust. You can sometimes still exploit flaws in these mechanisms to obtain a web shell for remote code execution. 
+
+
+## Table of Contents
+
+- [File Uploads: Unrestricted File Uploads!](#file-uploads-unrestricted-file-uploads)
+  - [Introduction](#Introduction)
+  - [Table of Contents](#table-of-contents)
+  - [How to identify](#how-to-identify)
+  - [Code/tools/websites](#codetoolswebsites)
+    - [Tools](#tools)
+    - [Websites](#websites)
+  - [Sample problem](#sample-problem)
+    - [Problem Statement](#problem-statement)
+    - [Solution](#solution)
+  - [References](#references)
+  - [Conclusion](#conclusion)
+
+
+## How to identify
+
+From a security perspective, the worst possible scenario is when a website allows you to upload server-side scripts, such as PHP, Java, or Python files, and is also configured to execute them as code. This makes it trivial to create your own web shell on the server. 
```

Comment:
From a security perspective, the worst possible scenario is when a website allows you to upload server-side scripts, such as PHP, Java, or Python files and is also configured to execute them as code. This makes it trivial to ~create~ upload your own web shell on the server.

Reply:
Can we have a short section here or in the intro article explaining what are web shells and a link to common ones used such as [this](https://pentestmonkey.net/tools/web-shells/php-reverse-shell) one